internal/config/env.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func (c *Config) addEnvVars() error {
+	authToken := os.Getenv("SQLC_AUTH_TOKEN")
+	if authToken != "" && !strings.HasPrefix(authToken, "sqlc_") {
+		return fmt.Errorf("$SQLC_AUTH_TOKEN doesn't start with \"sqlc_\"")
+	}
+	c.Cloud.AuthToken = authToken
+
+	return nil
+}
```

Comment:
I wanted to get away from calling `os.Getenv()` all over the place, but I'm not sure this (adding fields to `Config` struct and then populating after parse) is the best way to do it.

Reply:
I think as long as we only do this in one place, here in the config package, it's okay.